sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/TraceHelper.cs

```diff
@@ -111,7 +111,7 @@ internal static void AddActivityLinksToProperties(Activity activity, ref AzMonLi
                     {
                         if (MaxlinksAllowed < activity.Links.Count())
                         {
-                            AzureMonitorExporterEventSource.Log.WriteInformational("ActivityLinksIgnored", $"Max count of {MaxlinksAllowed} has reached.");
+                            AzureMonitorExporterEventSource.Log.ActivityLinksIgnored(MaxlinksAllowed, activity.DisplayName);
```

Comment:
Should we include `activity.Source.Name`?

Reply:
Do you think we need both? I thought at a minimum, collect the name of the activity.

```csharp
var activitySource = new ActivitySource("activitySourceName");
var activity = activitySource.StartActivity("activityDisplayName");
```

Reply:
added source name